### PR TITLE
ISPN-13521 cross-site expose type docs

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-13521

### CODE REVIEW

**Comment by oraNod** [comment]
@rvansa Hey Radim, I'm still a little unsure of this PR. It feels like it might be an appropriate level of detail for community docs but maybe not so much downstream. What are your thoughts?

I get the point is to provide guidance and not assume too much in terms of what users know about Kubernetes or OpenShift administration. At the same time we can't really take on the responsibility of providing guided procedures for provisioning the underlying infrastructure in our docs. 

Do you think we're hitting the right level of detail for the gap you identified when you were working with Tomas?

**Review by pruivo — APPROVED** [review]

**Review by rvansa — COMMENTED** [review]
I think that the level is detail is fine and I actually find community documentation pretty clear. The only think I would add is explicitly saying that when using managed replication the operator never modifies the remote cluster, the service account has read-only access.

In the new section in from this PR I would only add links to NodePort and LoadBalancer:
https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
https://docs.openshift.com/container-platform/4.9/networking/configuring_ingress_cluster_traffic/overview-traffic.html

File: documentation/asciidoc/topics/con_cross_site_expose_types.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[id='cross-site-expose-types_{context}']
+= Cross-site replication expose types
+
+[role="_abstract"]
+You can use a NodePort or a load balancer to handle network traffic for backup operations between {brandname} clusters.
```
[the remark this answers is not in the document]
Comment (oraNod) [reply documentation/asciidoc/topics/con_cross_site_expose_types.adoc:5]: I was going more for consistency with OpenShift docs here. They spell out load balancer in "regular English" but treat NodePort like an object. On second thought I'm actually going to spell out node port as two words to be consistent with the networking docs: https://infinispan.org/docs/infinispan-operator/main/operator.html#exposing-nodeport_network-services

[the remark this answers is not in the document]
Comment (oraNod) [reply documentation/asciidoc/topics/con_cross_site_expose_types.adoc:5]: Now that I read the Kubernetes docs again I can see they also use a mix of `NodePort`, `LoadBalancer`, and load balancer. In a couple of instances I can see they use node port as well.

I think when we talk about the value that you set for the expose type in `Infinispan` CR we should use `NodePort` or `LoadBalancer` but when we talk about the service in a general sense we should use "load balancer" and "node port".

**Comment by ryanemerson** [comment]
Thanks @oraNod

**Pull request merged**